generator/main.py

```diff
@@ -1,37 +1,95 @@
 import sys
-sys.path.append('ParseCAN')
-import ParseCAN
+import os
+import jinja2
+
+import ParseCAN.ParseCAN as ParseCAN
 
 import constants
-import pack_unpack_c
-import pack_unpack_h
-import enum_atom
-import structs
-import sys
 import computers_h
 import computers_c
-import test_h
-import test_c
-import send_receive
-import bus
+# import test_h
+# import test_c
 import drivers_inc
 
+from jinja2 import nodes
+from jinja2.ext import Extension
+from jinja2.exceptions import TemplateRuntimeError
+
+
+src_dir = '../src/'
```

Comment:
I would prefer pathlib be used for these. I see you have some instances of `{src_dir}/stuff` and some of `{src_dir}stuff`.

Reply:
`Path.join` would make this nicer and also work on other OSes.